lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.delete(2); arr.delete(3)'
```

Comment:
This works only if you know what values are in the array. We can use `pop` to remove the last element from an array no matter what is there.